README.md

```diff
@@ -1,2 +1,32 @@
-# examples
-Function examples for open-runtimes
+# Open Runtimes Functions Examples ⚡️
+
+![open-runtimes-box-bg-cover](https://user-images.githubusercontent.com/1297371/151676246-0e18f694-dfd7-4bab-b64b-f590fec76ef1.png)
+
+[![Discord](https://img.shields.io/discord/937092945713172480?label=discord&style=flat-square)](https://discord.gg/mkZcevnxuf)
+[![Twitter Account](https://img.shields.io/twitter/follow/appwrite?color=00acee&label=twitter&style=flat-square)](https://twitter.com/appwrite)
+
+Example Functions to show off what you can achieve with Open Runtimes.
+
+## List of Examples
+
+### Dart
+- [Welcome Email with Appwrite](/dart/welcome_email)
```

Comment:
I would suggest markdown table here. Ideally we would have the function in all runtimes, right?